src/bundle/Resources/views/content/relation.html.twig

```diff
@@ -1,12 +1,14 @@
 <td>
-    <a {% if not content.contentInfo.trashed %}
-        href="{{ path('_ezpublishLocation', { 'locationId': content.contentInfo.mainLocationId } ) }}"
-    {% endif %}>
+    {% if content.contentInfo.mainLocationId is not null %}
+        <a href="{{ path('_ezpublishLocation', { 'locationId': content.contentInfo.mainLocationId } ) }}">
+            {{ content.getName() }}
+        </a>
+    {% else %}
```

Comment:
What if mainLocationId is not null and content is trashed? Do we lose information that the content is in the trash?

Reply:
@mikadamczyk  https://github.com/ezsystems/ezplatform-admin-ui/pull/1353#discussion_r423662477 

If the content is trashed it has no mainLocationId assigned.

Reply:
@mikadamczyk What he says, but if we want to improve readability here for people not aware of this we could change logic to: `{% if published and mainLocationId is not null %}`